test/templates/config.ru

```diff
@@ -0,0 +1,63 @@
+require_relative "fake_middleware"
+
+$current_dir = File.dirname(__FILE__)
```

Comment:
> Also, look up the Pathname class in the standard lib.

OK :)

> And another question. Do we need it to point here, or the project root. E.g. current_dir vs current_project, both of them are used below.

Let's use the latter one. I think it could make sense to reduce `../` for the paths. The `project_root` seems better as naming?